HybrasylXML/XSD/Status.xsd

```diff
@@ -76,14 +83,15 @@
           <xs:attribute name="Id" type="xs:unsignedByte" use="required" />
         </xs:complexType>
       </xs:element>
+      <xs:element name="Messages" minOccurs="0" maxOccurs="1" type="status:Messages"/>
```

Comment:
Message effects? ???????

Reply:
A lot of basic statuses in usda have messages sent to the player. `The life is draining from your body` or `Poison is coursing through your veins`. Etc.

Reply:
I suppose that makes sense to have that tied in xml. OK